src/components/Login/Login.styled.js

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+
+const Container = styled.div.attrs(() => ({
```

Comment:
Same comment as before, I think you could avoid using `styled-components` when using Bootstrap and just have functions to compute classnames.

Reply:
Or do one or the other 😄